service-registry-cli/README.adoc

```diff
@@ -0,0 +1,327 @@
+
+.Procedure
+
+. Verify that you are using the Service Registry instance that you created.
++
+--
+This example switches to the `my-registry` instance that was previously created.
+
+.Selecting a Registry instance to use
+[source,shell]
+----
+$ rhoas service-registry use --name my-registry
+Service Registry instance "my-registry" has been set as the current instance.
+----
+--
+
+. Create a Service Registry artifact using example .
++
+--
+This example creates a Avro Schema artifact under `my-artifact` id and default group
+
+.Creating artifact
+[source,shell]
+----
+wget https://github.com/redhat-developer/app-services-cli/blob/main/docs/resources/avro-userInfo.json
+$ rhoas service-registry artifact create --type=AVRO --artifact-id=my-artifact avro-userInfo.json 
+----
+
+[NOTE]
+====
+You can setup custom values like --group,--version, --description, --name and --type to modify the artifact metadata when creating it.
+====
+--
+
+. If necessary, you can perform additional operation on artifacts like `rhoas service-registry artifact metadata-set`  
+
+. Manage a Service Registry artifacts.
++
+--
+This example we can list all artifacts for service registry and view their metadata
+
+.Listing artifact
+[source,shell]
+----
+rhoas service-registry artifact list 
+Using default artifacts group.
+  ID            NAME   CREATED ON                 CREATED BY             TYPE   STATE    
+ ------------- ------ -------------------------- ---------------------- ------ --------- 
+  my-artifact          2021-09-27T16:17:35+0000   wtrocki_kafka_devexp   AVRO   ENABLED  
+----
+
+Fetch their version information
+[source,shell]
+----
+rhoas service-registry artifact versions --artifact-id=my-artifact
+{
+  "count": 1,
+  "versions": [
+    {
+      "contentId": 29,
+      "createdBy": "wtrocki_kafka_devexp",
+      "createdOn": "2021-09-27T16:17:35+0000",
+      "globalId": 28,
+      "state": "ENABLED",
+      "type": "AVRO",
+      "version": "1"
+    }
+  ]
+}
+----
+
+and view metadata
+
+----
+rhoas service-registry artifact metadata-get --artifact-id=my-artifact
+Using default artifacts group.
+Fetching artifact metadata
+✔️ Successfully fetched artifact metadata
+{
+  "contentId": 29,
+  "createdBy": "wtrocki_kafka_devexp",
+  "createdOn": "2021-09-27T16:17:35+0000",
+  "globalId": 28,
+  "id": "my-artifact",
+  "modifiedBy": "wtrocki_kafka_devexp",
+  "modifiedOn": "2021-09-27T16:17:35+0000",
+  "state": "ENABLED",
+  "type": "AVRO",
+  "version": "1"
+}
+----
+
+--
+
+ 
+
+[discrete,id="commands-managing-registry_{context}"]
+=== Commands for managing Service Registry Instance
+
+The following tables show the `rhoas` commands you can use to manage your Service Registry instances and associated artifacts.
+For more information about any of the commands,
+view the command help (for example, `rhoas service-registry -h` or `rhoas service-registry artifact -h`).
+
+TODO ? Do we need that considering there is reference guide?
```

Comment:
Do we want to have the same list of commands like for Kafka.
I have feeling that we should be referencing users to guide that is out there rather than replicating list that is hard to maintain. We have it for kafka/CLI guide so it will also affect it.